godotsteam/godotsteam.h

```diff
@@ -35,6 +30,7 @@
 
 // Include GodotSteam headers
 #include "godotsteam_constants.h"

```

Comment:
I forgot godotsteam_constants.h has its own inclusion of the Steam header.  I might test moving it below and removing the inclusion.

Reply:
it has ifndef so where you move it doesnt matter, just set before you end up in constants.